### ISSUE TITLE
Feature: mongodb extension

### ISSUE THREAD

**Issue by mindmelting** [petergombos/cantrip#4]
Would be nice for this to work directly with mongodb.

Similar to Baucis, but without the need for schema definitions.

**Comment by borlaym** [comment]
At some point in the past the persistence layer was modular, and I did create implementations for mongodb, redis and even firebase (take a look at my repositories). We ultimately decided that cantrip shoud do one thing and do that well, but I image it would be possible to reintroduce that feature.

**Comment by jprichardson** [comment]
> We ultimately decided that cantrip shoud do one thing and do that well, 

I agree with this. I'd stick with it as principle. @mindmelting there are a number of MongoDB REST interfaces, why not just use those? It seems to me that cantrip's value proposition is that it's just JSON files.
